roles/lvm/files/mantl-storage-setup.py

```diff
@@ -0,0 +1,296 @@
+#!/usr/bin/env python
```

Comment:
Is a local copy to debug and development purposes,   controlable with  debug_storage_setup=True in ansible